### ISSUE TITLE
G6 - "Weather" icon has no text, just a link

### ISSUE THREAD

**Issue by jszasz** [brharp/hjckrrh#74]
Screen reader will only say its a link, does not have any text associated with it.

**Comment by jpinsent** [comment]
Weather icon now has associated link text "Weather" and is read by NVDA as "Guelph Weather".
